graviti/manager/draft.py

```diff
@@ -58,6 +69,40 @@ def __init__(
         self.updated_at = updated_at
         self.description = description
 
+    def _repr_head(self) -> str:
+        return f'{self.__class__.__name__}("{self.number}")'
+
+    def _get_factory_getter(self, offset: int, limit: int, sheet_name: str) -> Dict[str, Any]:
```

Comment:
should  `factory_getter` return `factory`?

Reply:
The return is the `factory_getter` instead of `factory`.